### PR TITLE
feat: add trade decline notification

### LINKED ISSUE #270: add trade decline notification

### PR DESCRIPTION
**Pull request by lfaivre**
note:

* closes #270

### CODE REVIEW

File: server/src/rooms/game/events/index.ts [hunk 1]
```diff
@@ -164,8 +164,8 @@ export class AcceptTradeRequest extends GameEventWithData {
       game.logs.push(log);
     }
     else{
-      game.players[game.tradeSet[this.data.id].to.role as Role].sendNotifcation('Trade partner cannot fulful trade.');
-      game.players[game.tradeSet[this.data.id].from.role as Role].sendNotifcation('A trade you cannot fulful has been deleted.');
+      game.players[game.tradeSet[this.data.id].to.role as Role].sendNotifcation('Trade partner cannot fulfill trade.');
```
Comment (alee) [inline server/src/rooms/game/events/index.ts]: typo here: should be `sendNotification`

File: server/src/rooms/game/events/index.ts [hunk 2]
```diff
@@ -184,7 +184,7 @@ export class RejectTradeRequest extends GameEventWithData {
     const toRole: Role = game.tradeSet[this.data.id].to.role;
     const fromRole: Role = game.tradeSet[this.data.id].from.role;
     
-    delete game.tradeSet[this.data.id];
+    game.removeTrade(this.data.id, 'reject-trade-request');
```
Comment (alee) [inline server/src/rooms/game/events/index.ts]: This is a good start. Can we push more logic into removeTrade like the notifications etc?

**Comment by alee** [comment]
I think these changes have been obsoleted by @chrstngyn removing notifications entirely from the app. I would like to cherry pick the `game.removeTrade(tradeId)` function and have that `game.log` what happened. 

If you're cool with that, can we close this PR @lfaivre ?

**Comment by lfaivre** [comment]
Yep, feel free!

**Comment by lfaivre** [comment]
closing this pr, addressed issues in #355

**Pull request closed without merging**